lib/express-rate-limit.js

```diff
@@ -62,11 +66,16 @@ function RateLimit(options) {
     setInterval(resetAll, options.windowMs);
 
     // export an API to allow hits from one or all IPs to be reset
+    function resetKey(key) {
+        delete hits[key];
+    }
 
+    // Backward compatibility function
     function resetIp(ip) {
-        delete hits[ip];
+        return resetKey(ip);
     }
 
+    rateLimit.resetKey = resetKey;
     rateLimit.resetIp = resetIp;
```

Comment:
You could just set `ratelimit.resetIp = resetKey;` and then get rid of the `resetIp()` function all together.